pkgs/development/tools/detect-secrets/default.nix

```diff
@@ -1,29 +1,52 @@
-{ lib, buildPythonApplication, fetchFromGitHub, isPy27, pyyaml, unidiff, configparser, enum34, future, functools32, mock, pytest }:
+{ lib
+, buildPythonApplication
+, configparser
+, enum34
+, fetchFromGitHub
+, functools32
+, future
+, isPy27
+, mock
+, pyahocorasick
+, pytestCheckHook
+, pyyaml
+, requests
+, responses
+, unidiff
+}:
 
 buildPythonApplication rec {
   pname = "detect-secrets";
-  version = "0.12.4";
+  version = "0.14.3";
+  disable = isPy27;
 
-  # PyPI tarball doesn't ship tests
```

Comment:
please, don't remove this comment, it refers about why we don't use the usual `fetchPypi` for fetching the source

Reply:
Sure, we can keep it.